kiwi_lint/one_to_one_field.py

```diff
@@ -0,0 +1,19 @@
+from pylint import interfaces
+from pylint import checkers
+from pylint.checkers import utils
+
+
+class OneToOneFieldChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'one-to-one-field-checker'
+
+    msgs = {'E4493': ("Do not use OneToOneField",
```

Comment:
Use `R4531` for error code and I will merge after #949 is merged.

Reply:
Fixed in 8a1c1cc